src/commands/cli/schemas/collect.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import * as os from 'os';
+import { FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { fs, Messages } from '@salesforce/core';
+import { cp } from 'shelljs';
+import * as fg from 'fast-glob';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'cli.schemas.collect');
+
+export default class Test extends SfdxCommand {
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+  public static readonly flagsConfig: FlagsConfig = {};
+
+  public async run(): Promise<void> {
+    const globs = ['node_modules/@sf/**/schemas/**/*.json', 'node_modules/@salesforce/**/schemas/**/*.json'];
```

Comment:
@mdonnalley  the `node_modules/@salesforce/**/schemas/**/*.json` with globstar after `@salesforce/` will recursively search for directories below that point for `/schemas/**/*.json`. Should this be `node_modules/@salesforce/*/schemas/**/*.json` ?